edocumentation-service/src/main/java/org/dcsa/edocumentation/service/mapping/DisplayedAddressMapper.java

```diff
@@ -45,16 +48,22 @@ private DisplayedAddress.DisplayedAddressBuilder setAddressLine(Integer index, S
 
   @SneakyThrows
   private List<String> getAddressLines(DisplayedAddress displayedAddress) {
-    List<String> addressLines = new ArrayList<>();
+    Map<Integer, String> addressLineMaps = new HashMap();
     for (Method m : displayedAddress.getClass().getMethods()) {
       if (m.getName().startsWith("getAddressLine") && m.getParameterTypes().length == 0) {
```

Comment:
How come we use reflection for this?

Reply:
it started out as quickly invoke all getAddressLineN methods, but I admit, adding the sorting to it made it more complex then invoking then directly one-by-one

Reply:
PS: I get it would work, but that looks fragile. I feel this logic belongs to  `DisplayedAddress` (e.g., via `toList()` or `asStringList()`).  Preferably without reflection, but if we do use reflection, it would at least be contained within object.

Reply:
changed to "direct" invocation